### PR TITLE
introducing `Storybook`

### PR DESCRIPTION
**Pull request by trdoan**

### CODE REVIEW

**Comment by trdoan** [comment]
@CongTan2661 Do you see this PR?

**Comment by trdoan** [comment]
@CongTan2661 Test Slack

**Comment by CongTan2661** [comment]
@trdoan Test Slack

**Comment by CongTan2661** [comment]
Test 1

**Comment by CongTan2661** [comment]
Test 2

**Comment by trdoan** [comment]
Test

**Comment by CongTan2661** [comment]
Test

**Comment by trdoan** [comment]
@CongTan2661  Ping !

File: package.json [hunk 1]
```diff
@@ -29,6 +31,15 @@
...
+    "@storybook/addon-essentials": "^7.0.5",
+    "@storybook/addon-interactions": "^7.0.5",
+    "@storybook/addon-links": "^7.0.5",
+    "@storybook/blocks": "^7.0.5",
+    "@storybook/nextjs": "^7.0.5",
+    "@storybook/react": "^7.0.5",
+    "@storybook/testing-library": "^0.0.14-next.2",
+    "@welldone-software/why-did-you-render": "^7.0.1",
+    "eslint-plugin-storybook": "^0.6.11",
+    "storybook": "^7.0.4"
```
Comment (trdoan) [inline package.json:43]: @CongTan2661 test comment

Comment (trdoan) [reply package.json:43]: @CongTan2661  Ping

Comment (trdoan) [reply package.json:43]: test

**Pull request closed without merging**